nixos/modules/services/torrent/transmission.nix

```diff
@@ -133,15 +133,13 @@ in
             description = "Executable to be run at torrent completion.";
           };
           options.umask = mkOption {
-            type = types.int;
-            default = 2;
+            type = types.str;
```

Comment:
This is a breaking change. Please consider if it is necessary. Also breaking changes have to be noted in release note.

Reply:
I don't know if it's needed, but I see no mention of that option accepting an int in the Transmission docs, and it's hard to tell if it works or not.